.circleci/verdaccio.yml

```diff
@@ -0,0 +1,44 @@
+storage: ./storage
+auth:
+  htpasswd:
+    file: ./htpasswd
+uplinks:
+  npmjs:
+    url: https://registry.yarnpkg.com/
+    max_fails: 40
+    maxage: 30m
+    timeout: 60s
+    fail_timeout: 10m
+    cache: false
+    agent_options:
+      keepAlive: true
+      maxSockets: 40
+      maxFreeSockets: 10
+packages:
+  # Group and isolate all local packages, avoid being proxy from outside
+  '@react-native/*':
+    access: $all
+    publish: $all
+  # The below specific entries can be removed once they are renamed and have the @react-native prefix
+  '@react-native-community/eslint-config':
+    access: $all
+    publish: $all
+  '@react-native-community/eslint-plugin':
+    access: $all
+    publish: $all
+  'react-native-codegen':
+    access: $all
+    publish: $all
+  'react-native-gradle-plugin':
+    access: $all
+    publish: $all
+  '@*/*':
+    access: $all
+    publish: $authenticated
+    proxy: npmjs
```

Comment:
Why is this needed?

Reply:
This prohibits unauthenticated users to published organization-scoped packages. I've seen this being done both in the Microsoft PR and the docusaurus one. @juanpicado, do you know the exact reasoning behind this?

Reply:
Great thanks for the clarification 👍